### ISSUE TITLE
Photo slider breaking pages

### ISSUE THREAD

**Issue by ldford** [Ecotrust/theredd#68]
When we add photos to existing slideshows (like the one halfway down the page here http://reddonsalmon.com/venue/main-hall/) the entire page breaks. We have to restore a previous version of the page. 

I see in plugins that there is an update available for Smart Slider (is that the plugin the in-page slideshows are using? I am not sure), but when I go to update it, it tells me I need license key. (see screenshot below) 

Basically we need to be able to add photos without breaking things.

![image](https://user-images.githubusercontent.com/6453589/50249418-d67aee80-0392-11e9-8d80-098401554ad0.png)

**Comment by ldford** [comment]
This is what it looks like when you try to add an image and the page breaks:

![image](https://user-images.githubusercontent.com/6453589/50249801-dd563100-0393-11e9-931e-e15f2da21073.png)

**Comment by ldford** [comment]
Wait @pollardld - I just did an update that was copy-only (no photo or photo slider involved) and it ALSO broke the entire page. So it maybe has nothing to do with the photos

**Comment by pollardld** [comment]
I think it has to do with Wordpress version 5 release. I am creating a dev environment now to test updating all the plugins and theme.

**Comment by pollardld** [comment]
I submitted a support request to Haven, asking for a Smart Slider Pro activation code. The plugin came with the theme. It would be pretty lame if they don't provide an activation key for us to use with their pre installed plugin. 

Everything but the Smart Slider Pro plugin has been updated on http://y0g.109.mwp.accessdomain.com/. There are a bunch of compatibility issues when updating the Haven theme. Separate from this issue, we should sort these bugs out before updating the Haven theme on reddonsalmon.com. 

Hopefully will hear back from themefuse soon with an activation key.

**Comment by ldford** [comment]
Thanks and also - that looks awful! Where do we even begin to sort out those bugs? It looks like it will take a full week of work....

**Comment by ldford** [comment]
And until we sort out the bugs and update the Haven theme, we cannot do any updates to reddonsalmon.com without breaking it. Do I have that right?

**Comment by pollardld** [comment]
ThemeForest responded to me with how to update the slider plugin. So I will see if that fixes things now.

For reference:  
Updates can be made from Appearance -> TF Plugin Updates
*Note: updates will begin automatically*

**Comment by pollardld** [comment]
All fixed. Fix was install the WP classic editor. The Haven theme looks to not be ready for the WP v5 editor. 

@ldford Please feel free to resume without free of breaking anything.
